compiler/lib/src/main/scala/codegen/CppWriter/AliasCppWriter.scala

```diff
@@ -120,27 +120,16 @@ case class AliasCppWriter (
       return linesMember(List())
     }
 
-    val systemHHeaders = List(
-      "FpConfig.h"
-    ).map(CppWriter.systemHeaderString).map(line)
-
     val standardHeaders = List(
       "Fw/Types/BasicTypes.h",
     ).map(CppWriter.headerString)
 
     val symbolHeaders = writeHIncludeDirectives(s, aNode)
     val headers = (standardHeaders ++ symbolHeaders).distinct.sorted.map(line)
-    linesMember(List.concat(
-      addBlankPrefix(systemHHeaders),
-      addBlankPrefix(headers)
-    ))
+    linesMember(addBlankPrefix(headers))
   }
 
   private def getHppIncludes: CppDoc.Member.Lines = {
-    val systemHppHeaders = List(
```

Comment:
I think we need to include BasicTypes.hpp here.

Reply:
OK, then I think we should include `Fw/Type/BasicTypes.hpp`. I think you are right, if the alias refers to a basic type, then the header for that specific type should be automatically included. Can we also add a comment explaining why we are using `Fw/Type/BasicTypes.hpp` instead of `Fw/FPrimeBasicTypes.hpp` here?